resolwe_bio/processes/modeling/pathway_predictor.yml

```diff
@@ -1,13 +1,16 @@
-- slug: pathway-predictor
+- slug: pathways-predictor
   name: Predict heterologous metabolic pathways
   requirements:
     expression-engine: jinja
     executor:
       docker:
         image: dddecaf/resolwe-pathway-predictor:latest
+    resources:
+      network: true
+      cores: 4
   data_name: "Predict heterologous metabolic pathways"
```

Comment:
You can construct a more informative data_name out of process inputs. [Example](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/go_enrichment/go.yml#L15) that includes gene names in the data_name